packages/bot/src/components/transactions.jsx

```diff
@@ -88,20 +88,23 @@ const Transaction = ({ contract }) => {
                         </div>
                     </td>
                     <td className='transactions__col'>
-                        <div
-                            className={className(
-                                'transactions__inline',
-                                'transactions__middle',
-                                [Math.sign(contract.profit) !== -1 ?
-                                    'transactions__green' :
-                                    'transactions__red'])}
-                        >
-                            <Money
-                                amount={Math.abs(contract.profit)}
-                                currency={contract.currency}
-                            />
-                        </div>
-
+                        {contract.profit ?
+                            <div
+                                className={className(
+                                    'transactions__inline',
+                                    'transactions__middle',
+                                    [Math.sign(contract.profit) !== -1 ?
```

Comment:
- I think it would be clearer to check whether profit is lower than 0 to decide which colour to show.
- You can use `classNames` more effective by just passing object args with conditions, see example below.

```JavaScript
className={classNames(
    'transactions__inline',
    'transactions__middle',
    {
        transactions__green: contract.profit > 0,
        transactions__red: contract.profit < 0
    })
}
```